SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLockingPolicy.h

```diff
@@ -17,17 +17,17 @@ class SPATIALGDK_API UAbstractLockingPolicy : public UObject
 	GENERATED_BODY()
 
 public:
-	virtual void Init(UAbstractStaticComponentView* InStaticComponentView, UAbstractPackageMapClient* InPackageMap, TSharedPtr<AbstractVirtualWorkerTranslator> InVirtualWorkerTranslator) {
+	virtual void Init(USpatialStaticComponentView* InStaticComponentView, UAbstractPackageMapClient* InPackageMap, TSharedPtr<AbstractVirtualWorkerTranslator> InVirtualWorkerTranslator) {
 		StaticComponentView = InStaticComponentView;
 		PackageMap = InPackageMap;
 		VirtualWorkerTranslator = InVirtualWorkerTranslator;
 	};
-	virtual ActorLockToken AcquireLock(AActor* Actor, FString LockName = "") PURE_VIRTUAL(UAbstractLockingPolicy::AcquireLock, return SpatialConstants::INVALID_ACTOR_LOCK_TOKEN;);
+	virtual ActorLockToken AcquireLock(AActor* Actor, FString LockName = "") PURE_VIRTUAL(UAbstractLockingPolicy::AcquireLock, return -1;);
```

Comment:
Why `-1`?

Reply:
Sorry, this was me trying to fix the really stupid bug 'm dealing with atm